src/modules/vendor/mixins/templateMixin.js

```diff
@@ -71,6 +83,25 @@ export const templateMixin = {
         NotifyNegative('Erreur lors de la mise à jour de la carte.');
       }
     },
+    async updateAnswer (index) {
+      try {
+        const newAnswer = get(this.card, `answers[${index}].label`) || '';
+        if (this.tmpInput === newAnswer) return;
+
+        if (index < 2) {
+          this.$v.card.answers.$each[index].label.$touch();
```

Comment:
juste `this.$v.card.answers.$each[index].$touch();` sans le label
et idem en dessous